src/components/passphrase/passphraseGenerator.test.js

```diff
@@ -29,15 +29,17 @@ describe('PassphraseGenerator', () => {
       const isTouchDeviceMock = mock(wrapper.instance()).expects('isTouchDevice');
       isTouchDeviceMock.returns(true);
       wrapper.instance().setState({}); // to rerender the component
-      expect(wrapper.find('.touch-fallback textarea')).to.have.lengthOf(1);
+      wrapper.update();
+      expect(wrapper.find('Input.touch-fallback textarea')).to.have.lengthOf(1);
     });
 
     it('shows at least some progress on pressing input if this.isTouchDevice()', () => {
       const wrapper = mount(<PassphraseGenerator {...props}/>);
       const isTouchDeviceMock = mock(wrapper.instance()).expects('isTouchDevice');
       isTouchDeviceMock.returns(true).twice();
       wrapper.instance().setState({}); // to rerender the component
-      wrapper.find('.touch-fallback textarea').simulate('change', { target: { value: 'random key presses' } });
+      wrapper.update();
+      wrapper.find('Input.touch-fallback textarea').simulate('change', { target: { value: 'random key presses' } });
```

Comment:
I see you've added tag name to css selector in multiple lines including this, would you please explain the problem?